frame/collective/src/lib.rs

```diff
@@ -215,6 +215,9 @@ pub mod pallet {
 
 		/// Weight information for extrinsics in this pallet.
 		type WeightInfo: WeightInfo;
+
+		/// Origin allowed to set collective members
+		type SetMembersOrigin:  EnsureOrigin<<Self as frame_system::Config>::RuntimeOrigin>;
```

Comment:
This MR will require a companion for Polkadot, since the Pallet is already deployed. Could you do that please?  
It is described [here](https://github.com/paritytech/substrate/blob/master/docs/CONTRIBUTING.adoc#updating-polkadot-as-well).